docs/src/main/tut/dsl.md

```diff
@@ -86,7 +86,17 @@ But here in the REPL, it's up to us to run it:
 val response = io.unsafeRunSync
 ```
 
-Cool.
+Cool.  Now we will be able to test things like the status:
+
+```tut
+val status = response.status
+```
+
+Or the content of our body:
+
+```tut
+val body = response.bodyAsText.compile.toList.unsafeRunSync().mkString("")
```

Comment:
How about just `response.as[String].unsafeRunSync()`?

Reply:
Will try that out, that looks a lot better imo

Reply:
And now that I try it again, I naturally cannot reproduce... strange, wonder if I did a typo or if I hit some kind of corner case :S